curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/63507e4562cdde3a28e8de1b.md

```diff
@@ -29,10 +29,10 @@ Your `.filter()` method should have a callback which accepts `el` as a parameter
 assert.match(calculate.toString(), /numbers\.filter\(\(?\s*el\s*\)?\s*=>|numbers\.filter\(function\s*\(?el\)\s*\{/)
 ```
 
-Your callback function should use `!` and `Number.isNaN()` to check if `el` is NOT `NaN`.
+Your callback function should use `!` and `isNaN()` to check if `el` is NOT `NaN`.
 
 ```js
-assert.match(calculate.toString(), /!(Number\.)?isNaN\(\s*el\s*\)/);
+assert.match(calculate.toString(), /!\s*isNaN\(\s*el\s*\)/);
```

Comment:
```suggestion
assert.match(calculate.toString(), /!\s*(Number\.)?isNaN\(\s*el\s*\)/);
```

I'm not sure we should completely remove the possibility of prefixing with `Number.`. It is perfectly legal to use it without `Number.` and probably most people will and that's what we'll teach, but it is also legal to use it as `Number.isNaN` and most likely at some point someone who knows this will try it and if the tests don't accept it then we'll hear about it in the forum :slightly_smiling_face:

Reply:
True! You are very correct